packages/application-shell/src/components/quick-access/constants.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/prefer-default-export
+
+export const permissions = {
+  ViewOrders: 'ViewOrders',
+  ManageOrders: 'ManageOrders',
+  ViewProducts: 'ViewProducts',
+  ManageProducts: 'ManageProducts',
+  ViewCategories: 'ViewCategories',
+  ManageCategories: 'ManageCategories',
+  ViewCustomers: 'ViewCustomers',
+  ManageCustomers: 'ManageCustomers',
+  ViewCustomerGroups: 'ViewCustomerGroups',
+  ManageCustomerGroups: 'ManageCustomerGroups',
+  ViewProductDiscounts: 'ViewProductDiscounts',
+  ManageProductDiscounts: 'ManageProductDiscounts',
+  ViewDiscountCodes: 'ViewDiscountCodes',
+  ManageDiscountCodes: 'ManageDiscountCodes',
+  ViewCartDiscounts: 'ViewCartDiscounts',
+  ManageCartDiscounts: 'ManageCartDiscounts',
+  ViewProjectSettings: 'ViewProjectSettings',
+  ManageProjectSettings: 'ManageProjectSettings',
+  ViewDeveloperSettings: 'ViewDeveloperSettings',
+  ManageDeveloperSettings: 'ManageDeveloperSettings',
+  ViewProductTypes: 'ViewProductTypes',
+  ManageProductTypes: 'ManageProductTypes',
+};
```

Comment:
So not to depend on the permissions package anymore. Given we add more permissions there this can be removed again but I'd prefer not to do that now.